src/Sulu/Bundle/ContactBundle/Tests/Functional/Controller/AccountMediaControllerTest.php

```diff
@@ -26,45 +26,24 @@
 
 class AccountMediaControllerTest extends SuluTestCase
 {
-    /**
-     * @var KernelBrowser
-     */
-    private $client;
+    private KernelBrowser $client;
```

Comment:
We should do this kind of refactoring in own pull request. Else this kind of pull requests get just unnecessary big to review

Reply:
Reverted